src/transport/heat_model.cc

```diff
@@ -238,9 +234,26 @@ IT::Selection HeatTransferModel::ModelEqData::get_output_selection_input_type(co
 }
 
 
-HeatTransferModel::HeatTransferModel() :
-		flux_changed(true)
-{}
+HeatTransferModel::HeatTransferModel(Mesh &mesh, const Input::Record in_rec) :
```

Comment:
Why this constructor do principally more tasks then the constructor of the ConcentrationTransport model?